ui/src/main/java/com/epam/eco/kafkamanager/ui/topics/browser/handlers/FilterOperationHandler.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ *  Copyright 2023 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License.  You may obtain a copy
+ *  of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ *******************************************************************************/
+package com.epam.eco.kafkamanager.ui.topics.browser.handlers;
+
+import com.epam.eco.kafkamanager.FilterClause;
+
+import static java.util.Objects.isNull;
+
+public abstract class FilterOperationHandler<T> {
+
+    public static final String KEY_VALUE_SEPARATOR = ":";
+    public static final String PROPERTIES_SEPARATOR = ".";
+    private final FilterClause clause;
+
+    public FilterOperationHandler(FilterClause clause) {
+        this.clause = clause;
+    }
+
+    public FilterClause getClause() {
+        return clause;
+    }
+
+    public boolean compare(T value) {
```

Comment:
Why I should return here a number. Main idea of this "naming convention" is similarity of the filter options and method names, so I think that boolean value here reasonable. Because it just similar method as e.q. equals in filter notation.

Reply:
I just meant, compare() in the Java ecosystem is expected to return the number, but here it returns Boolean instead. It's a bit confusing, though.

Reply:
I am understand it, but as long as it is very difficult to understanding part of code I decide name this methods as like as in filters